docs/providers/aws/events/http-api.md

```diff
@@ -283,3 +283,15 @@ provider:
   httpApi:
     metrics: true
 ```
+
+### Tags
+
+With HTTP API we can tag gateway.
```

Comment:
Let's maybe expand that documentation a little bit more. Maybe something like:


When using HTTP API, it is possible to tag the corresponding API Gateway. By setting `provider.httpApi.useProviderTags` to `true`, all tags defined on `provider.tags` will be applied to API Gateway.

```yaml
provider:
  tags:
    project: myProject
  httpApi:
    useProviderTags: true
```

In the above example, the tag `project: myProject` will be applied to API Gateway. 

Note: If the API Gateway has any existing tags applied outside of Serverless Framework, they will be removed during deployment.

Reply:
just pushed, thanks for the detailed comment and sorry for my wording